suite-native/module-accounts-import/src/screens/SelectCoinScreen.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+
+import {
+    AccountsImportStackParamList,
+    AccountsImportStackRoutes,
+    Screen,
+    StackProps,
+} from '@suite-native/navigation';
+import { mainnets, mainnetsOrder, testnets, testnetsOrder } from '@suite-native/config';
+import { Box } from '@suite-native/atoms';
+
+import { updateSelectedCoin } from '../accountsImportSlice';
+import { AccountImportHeader } from '../components/AccountImportHeader';
+import { AssetItem, SelectableAssetList } from '../components/SelectableAssetList';
+import { AccountImportTitle } from '../components/AccountImportTitle';
+
+export const SelectCoinScreen = ({
+    navigation,
+}: StackProps<AccountsImportStackParamList, AccountsImportStackRoutes.SelectCoin>) => {
+    const dispatch = useDispatch();
+
+    const handleSelectCoin = (assetItem: AssetItem) => {
+        dispatch(updateSelectedCoin(assetItem));
+        navigation.navigate(AccountsImportStackRoutes.XpubScan, {});
+    };
+
+    return (
+        <Screen header={<AccountImportHeader activeStep={1} />}>
+            <Box>
```

Comment:
Isn't the box useless here?

Reply:
removed e21e5713ac754d22544aedeb5ae901093cf708a7